pkg/storage/snapshot/snapshot_test.go

```diff
@@ -1199,13 +1199,13 @@ var _ = Describe("Snapshot controlleer", func() {
 				testutils.ExpectEvent(recorder, "SuccessfulVolumeSnapshotCreate")
 			})
 
-			It("should update VirtualMachineSnapshotContent", func() {
+			DescribeTable("should update VirtualMachineSnapshotContent", func(readyToUse bool) {
```

Comment:
what about testing the unfreeze is not called before "created" and is being called after it

Reply:
good idea